WalletWasabi.Gui/Styles/Styles.xaml

```diff
@@ -28,6 +28,10 @@
     </Style.Resources>
   </Style>
 
+  <Style Selector="TabStripItem">
```

Comment:
can you add:
`xmlns:shell="clr-namespace:AvalonStudio.Controls;assembly=AvalonStudio.Shell.Extensibility"`
and make the selector target:

`shell|DocumentTabControl /template/ TabStripItem`

that way this wont apply to any other controls in the app except the one you wish to change.

Reply:
done, though I think it might look good elsewhere, but I didn't find any other occurrences to test with :)

Reply:
As pointed out by @lontivero (I didn't notice this before sorry about that) this causes duplicate issues.
Even if I try to map it to a different name such as xmlns:ext="clr-namespace:AvalonStudio.Controls;assembly=AvalonStudio.Shell.Extensibility" and add ext|DocumentTabControl /template/ to the selector, it won't build.
As mentioned above, I'm not convinced that limiting this is necessary, as it's most likely an issue anywhere the TabStripItem is used. Do you think this is a must? If so can you point out where this affects other controls within the app?